tests/acceptance/stepDefinitions/loginContext.js

```diff
@@ -59,6 +59,10 @@ Then('the authentication page should be visible', () => {
   return loginPage.waitForElementPresent('@authenticateButton')
 })
 
+Then('the redirected url should contain {string}', function(urlContent) {
```

Comment:
This step can be used for any URL, not only the "redirected url"
```suggestion
Then('the user should be on page with the url containing {string}', function(urlContent) {
```